src/main/java/org/mitre/synthea/world/agents/Person.java

```diff
@@ -83,6 +84,7 @@ public Person(long seed) {
     attributes = new ConcurrentHashMap<String, Object>();
     vitalSigns = new ConcurrentHashMap<VitalSign, Double>();
     symptoms = new ConcurrentHashMap<String, Map<String, Integer>>();
+    symptomStatuses = new ConcurrentHashMap<String, Map<String, Boolean>>();
```

Comment:
I think it would be ideal to have a real Symptom class that was associated to the Conditions, but I'm ok with this as a first step.